examples/source/1.components/amp-bind-macro.html

```diff
@@ -0,0 +1,57 @@
+<!---
+
+author: elisameyer, philkrie
+
+--->
+<!--
+  ## Introduction
+
+  This example shows how you can sanitize user input for a URL with an
+  `amp-bind-macro` call.
+-->
```

Comment:
add an empty comment afterwards to avoid that the doctype will be displayed: 

```
		   This example shows how you can sanitize user input for a URL with an
   `amp-bind-macro` call.
 -->
<!-- -->
```

Reply:
updated